src/istio/zarf.yaml

```diff
@@ -39,11 +56,38 @@ components:
     charts:
       - name: istiod
         valuesFiles:
-          - "values/registry1-values.yaml"
+          - "values/registry1/istiod.yaml"
     images:
       - registry1.dso.mil/ironbank/tetrate/istio/proxyv2:1.24.2-tetratefips0
       - registry1.dso.mil/ironbank/tetrate/istio/pilot:1.24.2-tetratefips0
 
+  - name: istio-ambient
+    required: false
+    only:
+      flavor: registry1
+    import:
+      path: ambient
+    charts:
+      - name: cni
+        valuesFiles:
+          # Update once Registry1 has working cni image
+          #- "values/registry1/cni.yaml"
+          - "values/upstream/cni.yaml"
+      - name: ztunnel
+        valuesFiles:
+          # Update once Registry1 has working ztunnel image
+          #- "values/registry1/ztunnel.yaml"
+          - "values/upstream/ztunnel.yaml"
+    images:
+      # Registry1 ztunnel image is not working right now
+      # https://repo1.dso.mil/dsop/tetrate/istio/1.24/ztunnel/-/issues/9
+      #- registry1.dso.mil/ironbank/tetrate/istio/ztunnel:1.24.2-tetratefips0
+      - docker.io/istio/ztunnel:1.24.2-distroless
+      # Registry1s install-cni-fips is not working right now
+      # https://repo1.dso.mil/dsop/tetrate/istio/1.24/install-cni/-/issues/11
+      #- registry1.dso.mil/ironbank/opensource/istio/install-cni:1.24.2
+      - docker.io/istio/install-cni:1.24.2-distroless
```

Comment:
Might be good to make sure we have an issue tracking these todos to switch over - esp since the original issues got closed out (not sure all the problems are solved though!)